crates/iota/src/name_commands.rs

```diff
@@ -972,23 +1090,30 @@ impl SubdomainCommand {
                 )
                 .await?;
 
-                NameCommandResult::Client(
-                    IotaClientCommands::Call {
-                        package: subdomains_package,
-                        module: "subdomains".to_owned(),
-                        function: "extend_expiration".to_owned(),
-                        type_args: Default::default(),
-                        args: vec![
-                            IotaJsonValue::from_object_id(iota_names_config.object_id),
-                            IotaJsonValue::from_object_id(nft.id()),
-                            IotaJsonValue::new(JsonValue::Number(expiration_timestamp.0.into()))?,
-                        ],
-                        gas_price: None,
-                        opts: opts.clone(),
-                    }
-                    .execute(context)
-                    .await?,
-                )
+                let res = IotaClientCommands::Call {
+                    package: subdomains_package,
+                    module: "subdomains".to_owned(),
+                    function: "extend_expiration".to_owned(),
+                    type_args: Default::default(),
+                    args: vec![
+                        IotaJsonValue::from_object_id(iota_names_config.object_id),
+                        IotaJsonValue::from_object_id(nft.id()),
+                        IotaJsonValue::new(JsonValue::Number(expiration_timestamp.0.into()))?,
+                    ],
+                    gas_price: None,
+                    opts: opts.clone(),
+                }
+                .execute(context)
+                .await?;
+
+                handle_transaction_result(res, async |res| {
+                    Ok(NameCommandResult::ExtendExpiration {
+                        nft: get_owned_nft_by_name::<SubdomainRegistration>(&domain, context)
```

Comment:
I wonder if we also want to print the entry since the expiration is also there (assuming it's changed in both)

Reply:
🤷‍♀️ up to you I guess